sql/core/src/test/scala/org/apache/spark/sql/DataFrameFunctionsSuite.scala

```diff
@@ -110,7 +110,20 @@ class DataFrameFunctionsSuite extends QueryTest {
       testData2.collect().toSeq.map(r => Row(~r.getInt(0))))
   }
 
-  test("length") {
+  test("if function") {
```

Comment:
how about put the function in the front. e.g.
`function if`, `function length` etc. Or
`conditional function if`
`string function length`
`string function trim`.. ?